astropy/coordinates/transformations.py

```diff
@@ -509,17 +487,13 @@ def to_dot_graph(
         dotgraph : str
             A string with the DOT format graph.
         """
-        nodes = []
-        # find the node names
-        for a in self._graph:
-            if a not in nodes:
-                nodes.append(a)
-            for b in self._graph[a]:
-                if b not in nodes:
-                    nodes.append(b)
-        for node in addnodes:
-            if node not in nodes:
-                nodes.append(node)
+        # We store nodes as `dict` keys because differently from `list` uniqueness is
```

Comment:
I guess this can be factored out, but probably best not for this reorganization.